PR: Add FAQ question on installing Spyder on macOS Big Sur

CAM-Gerlach opened the pull request:
<!--- Before submitting your pull request, --->
<!--- please complete as much as possible of the following checklist: --->

### Pull Request Checklist

* [x] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
* [x] Based your PR on the latest version of the correct branch (master or 3.x)
* [x] Checked your writing carefully for correct English spelling, grammar, etc
* [x] Described your changes and the motivation for them below
* [x] Noted what issue(s) this pull request resolves, creating one if needed



## Description of Changes

<!--- Describe what you've changed and why. --->

Adds a FAQ question describing the macOS Big Sur situation and explaining how to work around it, to be linked to from the website download section (once a PR that implements #204 is implemented).

![image](https://user-images.githubusercontent.com/17051931/100554184-9a2bba80-3258-11eb-9e50-7ff1eb5c58ac.png)



### Issue(s) Resolved

<!--- Pull requests should typically resolve at least one—preferably only one—
<!--- outstanding issue; create a new one if no relevant issue exists.
<!--- List the issue(s) below, in the form "Fixes #1234" . One per line.--->

Fixes #203 


<!--- Thanks for your help making Spyder --->
<!--- and its documentation better for everyone! --->

juanis2112 commented on doc/faq.rst:
```diff
@@ -281,3 +281,24 @@ Troubleshooting
+
+.. dropdown:: Q: Spyder doesn't launch or is slow on macOS 11 Big Sur. How can I get it working?
+
+   Spyder is in the final stages of being updated for full compatibility with macOS Big Sur, which will be released in version 4.2.1, due out before the end of 2020.
```
```suggestion
   Spyder is in the final stages of being updated for full compatibility with macOS Big Sur, which will be released in version 4.2.1, before the end of 2020.
```

ccordoba12 replied:
```suggestion
   Spyder is in the final stages of being updated for full compatibility with macOS Big Sur, which will be part of version 4.2.1, to be released before the end of 2020.
```

CAM-Gerlach replied:
```suggestion
   Spyder is in the final stages of being updated for full compatibility with macOS Big Sur, which will be released by the end of 2020 as part of version 4.2.1.
```

Started with Carlos' version to avoid the grammar/syntax problems with the other and reversed the order of the clauses, to avoid the complex/choppy phrasing present from my original version.

ccordoba12 commented on doc/faq.rst:
```diff
@@ -281,3 +281,24 @@ Troubleshooting
+
+   Spyder is in the final stages of being updated for full compatibility with macOS Big Sur, which will be released in version 4.2.1, due out before the end of 2020.
+   However, you can get it working right now with the workaround below.
+   Make sure you have the Anaconda or Miniconda distribution installed, and run the following commands in the Terminal to install Spyder in a clean environment:
```
```suggestion
   Make sure you have the Anaconda or Miniconda distribution installed, and run the following commands in the Terminal to install Spyder from Conda-forge in a clean environment:
```

CAM-Gerlach replied:
```suggestion
   Make sure you have the Anaconda or Miniconda distribution installed, and run the following commands in the Terminal to install Spyder from Conda-Forge in a clean environment:
```

Use consistent capitalization

ccordoba12 reviewed (commented):
Thanks @CAM-Gerlach for this! Two small suggestions, otherwise looks good to me.

CAM-Gerlach commented:
Thanks for the reviews. I'll apply the tweaked suggestions and then rebase them in plus the compat tweak for PR #207 . Standby.

CAM-Gerlach commented:
@juanis2112 @ccordoba12 I applied the changes and added the class id for PR #207 , rebased and fixuped everything and tested it, so this should be ready to go.

ccordoba12 reviewed (approved):
Looks good to me, thanks @CAM-Gerlach!

ccordoba12 commented:
I'm going to merge this one because it's important to have it as soon as possible.

CAM-Gerlach commented:
@ccordoba12 Okay, I'll go ahead immediately with the PR to add a link to it on the downloads section of the website.

ccordoba12 commented:
Thanks!

CAM-Gerlach commented:
Opened as spyder-ide/website-spyder#190 , ready to go when you are
